devs/docs/benchmarks.rst

```diff
@@ -16,7 +16,7 @@ This will execute all benchmarks which are in ``benchmarks/src/test/java``.
 
 If you want to execute specific benchmarks you can provide a filter argument::
 
-    $ java -jar benchmarks/build/libs/crate-benchmarks-all.jar <benchmarkMethodName>
+    $ java -jar benchmarks/build/libs/crate-benchmarks-all.jar <benchmarkClassName[.methodName]>
```

Comment:
Specifying only method names works as well. So I think it would be `[ methodName | className[.methodName] ]`  (and the commit msg is a bit misleading)